components/script/dom/window.rs

```diff
@@ -1374,8 +1374,8 @@ impl Window {
         issued_reflow
     }
 
-    pub fn layout_reflow(&self, query_msg: QuerMsg) -> bool {
-        self.reflow(ReflowGoal::LayoutQuery(query_msg, time::precise_time_ns()), ReflowReason::Query)
+    pub fn layout_reflow(&self, query_msg: QueryMsg) -> bool {
+        self.reflow(ReflowGoal::LayoutQuery(query_msg, 0u64), ReflowReason::Query)
```

Comment:
We do actually care about the timestamp of when the query was created in this thread.